### PR TITLE
Create how-to-add-tech-alliance.md

### PR DESCRIPTION
**Pull request by jasonblais**
@justinegeffen Pinging your help with editor review (but can't add you as a reviewer, have asked team to take a look at permissions for this repo)

### CODE REVIEW

File: operations/messaging-and-math/how-to-guides-for-m-and-m/how-to-add-tech-alliance.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+description: 50% Draft
+---
+
+# How to add new tech alliance to https://mattermost.com/partners/
```
Comment (justinegeffen) [inline operations/messaging-and-math/how-to-guides-for-m-and-m/how-to-add-tech-alliance.md]:
```suggestion
# How to add a new tech alliance to https://mattermost.com/partners/
```

File: operations/messaging-and-math/how-to-guides-for-m-and-m/how-to-add-tech-alliance.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+1. Ask tech alliance to provide the following information to partner-directory@mattermost.com:
```
Comment (justinegeffen) [inline operations/messaging-and-math/how-to-guides-for-m-and-m/how-to-add-tech-alliance.md]:
```suggestion
1. Ask the tech alliance to provide the following information in an email to partner-directory@mattermost.com:
```

File: operations/messaging-and-math/how-to-guides-for-m-and-m/how-to-add-tech-alliance.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+ - Partner Name
+ - Partner Contact Information
+ - Partner Description
+ - City, State/Province and Country
+ - Logo 
+
+2. Sales Ops reviews the information, and either contacts the partner for additional details or forwards the information to Marketing
```
Comment (justinegeffen) [inline operations/messaging-and-math/how-to-guides-for-m-and-m/how-to-add-tech-alliance.md]:
```suggestion
2. Sales Ops reviews the information, and either contacts the partner for additional details or forwards the information to Marketing: 
```

Comment (justinegeffen) [reply operations/messaging-and-math/how-to-guides-for-m-and-m/how-to-add-tech-alliance.md]: Do all contacts get emailed or is it an alias email address for each team?

File: operations/messaging-and-math/how-to-guides-for-m-and-m/how-to-add-tech-alliance.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+ - Sales Ops contacts: Melissa Warner, Elaine Yue
+ - Marketing contact: Lauren Nguyen
+
+3. Marketing team creates an Asana task [in the Web Dev Asana board](https://app.asana.com/0/1118355038043647/board) and adds the partner information to https://mattermost.com/partners/
```
Comment (justinegeffen) [inline operations/messaging-and-math/how-to-guides-for-m-and-m/how-to-add-tech-alliance.md]:
```suggestion
3. Marketing team creates an Asana task [in the Web Dev Asana board](https://app.asana.com/0/1118355038043647/board) and adds the partner information to https://mattermost.com/partners/. 
```

**Comment by justinegeffen** [comment]
General comment: The doc refers to tech alliance and also to partner - are they the same? Apologies if missing context around the doc.

File: operations/messaging-and-math/how-to-guides-for-m-and-m/how-to-add-tech-alliance.md (hunk 3 above)
Comment (jasonblais) [reply operations/messaging-and-math/how-to-guides-for-m-and-m/how-to-add-tech-alliance.md]: I wasn't sure what you meant by

> Do all contacts get emailed or is it an alias email address for each team?

**Comment by jasonblais** [comment]
@justinegeffen It's synonymous - if we changed the title of the doc to "How to add a new tech alliance partner to https://mattermost.com/partners/", would it be more clear?

Also, thanks for the suggestions, all great callouts

**Comment by jasonblais** [comment]
@justinegeffen Ready for another review

File: operations/messaging-and-math/how-to-guides-for-m-and-m/how-to-add-tech-alliance.md (hunk 3 above)
Comment (justinegeffen) [reply operations/messaging-and-math/how-to-guides-for-m-and-m/how-to-add-tech-alliance.md]: You can disregard that - I misread the section and on the second review realizes that it's a list of the contacts in those teams.

**Comment by justinegeffen** [comment]
> @justinegeffen Ready for another review

Thanks, @jasonblais, for the changes. It looks good to me. :)

**Pull request merged**